### ISSUE TITLE
Class-balanced CE loss function leads to worse result?

### ISSUE THREAD

**Issue by rosefun** [LeeJunHyun/Image_Segmentation#10]
I find that the class-balanced cross entropy loss function leads to smaller loss, about 0.001 - 0.008 in my dataset, and the performance is worse than using Iou as loss function which has bigger loss value .

**Comment by LeeJunHyun** [comment]
Hi, @rosefun .
I've never used class-balanced CE loss function, and also I don't know what is your dataset.
So I have no idea about that :(
Can you check class distribution of your dataset?

**Comment by LeeJunHyun** [comment]
If you don't have further issues, I will close the issue. I hope my answer was helpful :)
